hilti/toolchain/include/ast/attribute.h

```diff
@@ -94,6 +94,7 @@ const Kind Nosub("&nosub");
 const Kind OnHeap("&on-heap");
 const Kind Optional("&optional");
 const Kind Priority("&priority");
+const Kind PublicSignature("&public-signature");
```

Comment:
Wondering if we could just call this `&public`, aligning it with the `public` linkage that has similar semantics. (unless you guys discussed that already?)

Reply:
`&public` makes sense to me. I think I initially had it as `&keep-signature`, so @bbannier suggested `&public-signature`. Once it's `&public-signature` it makes sense to just call it `&public` though - but took a couple iterations to get there